cpp/tests/io/nested_json_test.cpp

```diff
@@ -148,6 +148,11 @@ TEST_F(JsonTest, StackContext)
   auto const stream = cudf::get_default_stream();
 
   // Test input
+  std::array<char, 4> delimiter_chars{{'\n', '\b', '\v', '\f'}};
```

Comment:
I had to modify both the `StackContext` and `StackContextUTF8` tests because `get_stack_context` now accepts a delimiter character. Both tests are run with `PushPopWithoutReset` so the delimiter character that is passed to `get_stack_context` should not affect the correctness of the test, so there is no change in coverage. 

Having a randomly chosen delimiter might be excessive - we can also just hard-code with an arbitrarily chosen character.

Reply:
> Having a randomly chosen delimiter might be excessive - we can also just hard-code with an arbitrarily chosen character.

Let's do that